data/course-surveys.csv

```diff
@@ -0,0 +1,25 @@
+year,Timestamp,How would you rate the course *lecture* material overall?,"How can the course material be improved (for example, the pacing, structure, content, concepts, and/or challenges)?",The ratio of instructors to students was:,How would you rate the instructors overall?,What can the instructors improve upon?,How would you rate the final project?,How could the final project be improved?,Which were the biggest differences between the final project in this class and your previous undergraduate group projects?,Did the lectures prepare you adequately for the final project?,How relevant do you think the concepts discussed during class will be for your future work/studies?,Did you need to spend more or less time and effort on this course than on other courses?,Is there anything specific we could do to help make this course better for future semesters?,What did you enjoy the most about the course and why?,"Were there any skills that you developed more in this course compared to your previous courses? If so, which ones?",Was there anything you had hoped/expected to learn starting out in this course that we did not cover?,Would you recommend this course to a friend?,How would you rate the course overall?
```

Comment:
Could you make the columns easier to read/use? (e.g. "Date", "OverallRating", "ImprovementsFeedback", etc)